new_frontend/src/pages/weekly/index.tsx

```
@@ -44,24 +61,31 @@ function StudentProfileWithFilters({ student }: { student: Student }) {
       image={student.image}
-      name={student.name}
       student_name={student.student_name}
-      classSection={student.classSection}
       reference_number={student.reference_number}
       custom_division={student.custom_division}
       first_name={student.first_name}
       last_name={student.last_name}
       program_name={student.program_name}
       isLoading={classLoading}
-      children={<WeeklyChildren name={student.first_name} division={classDetails.division.name} />}
+      children={<WeeklyChildren name={student.first_name} division={classDetails.division.name} studentId={student.name} />}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)